build/azure-pipelines/ci-build-test.yml

```
@@ -0,0 +1,152 @@
+
+stages:
+- stage: BuildAndTest
+  jobs:
+
+  - job: Ubuntu
+    displayName: 'Ubuntu Agent'
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    steps:
+    - task: UseDotNet@2
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)